pkg/resource/statefulset.go

```diff
@@ -191,6 +213,24 @@ func (b StatefulSetBuilder) makePodTemplate() corev1.PodTemplateSpec {
 	return pod
 }
 
+// MakeInitContainers creates a slice of corev1.Containers which includes a single
+// corev1.Container that is based on the CR.
+func (b StatefulSetBuilder) MakeInitContainers() []corev1.Container {
+	initContainer := fmt.Sprintf("%s-init", DbContainerName)
+	return []corev1.Container{
+		{
+			Name:            initContainer,
+			Image:           ubiimage,
```

Comment:
what container image are we using here? I thought we decided to use either the db image or the operator image but I can't tell from this.